shell/common/node_bindings.cc

```diff
@@ -488,6 +494,7 @@ node::Environment* NodeBindings::CreateEnvironment(
     // in renderer processes this should be blink. We need to tell Node.js
     // not to register its handler (overriding blinks) in non-browser processes.
     flags |= node::EnvironmentFlags::kNoRegisterESMLoader |
+             node::EnvironmentFlags::kNoBrowserGlobals |
```

Comment:
`_noBrowserGlobals` usage in this function should be deleted in favor of this flag.